homeassistant/components/media_player/__init__.py

```diff
@@ -310,7 +310,9 @@ async def async_setup(hass, config):
     component.async_register_entity_service(
         SERVICE_PLAY_MEDIA,
         vol.All(
-            cv.make_entity_service_schema(MEDIA_PLAYER_PLAY_MEDIA_SCHEMA),
+            cv.make_entity_service_schema(
+                MEDIA_PLAYER_PLAY_MEDIA_SCHEMA, extra=vol.ALLOW_EXTRA
```

Comment:
This is not a good idea. Instead add a new `extra` key to the schema

Reply:
The core `async_play_media` and `play_media` methods already accept and pass down `**kwargs` to the integration implementations: https://github.com/home-assistant/core/blob/dev/homeassistant/components/media_player/__init__.py#L602-L606

Should I then just pass `**kwargs["extra"]` into the pychromecast call?

Reply:
Yes

Reply:
Testing change now, will push soon.